desktop/apps/auction2/test/artwork.js

```diff
@@ -0,0 +1,26 @@
+require('babel-core/register')
+require('coffee-script/register')
```

Comment:
👍 I think we needed these originally when we were using es6 for the first time here, but looks like they are now unneeded. Removed from this and the other tests here!